src/compiler/get_names.ts

```diff
@@ -41,6 +41,8 @@ export async function getNamesBeforeCli() {
         .toString()
         .trim();
 
+    const nativeCompilation = process.argv.includes('--native-compilation');
```

Comment:
Could you create an issue to add `azle --help` to give a simple overview of all of the flags azle has?